keyboards/melgeek/z70ultra/z70ultra.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+#define LAYOUT( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, \
+    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, \
+    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B,      K2D, \
+    K30,      K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C,      \
+    K40, K41, K42,      K44, K45, K46,           K49, K4A,                \
+	                              K56, K57, K58, K59, K5A, K5B, K5C, K5D  \
+) { \
+    { K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D }, \
+    { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D }, \
+    { K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, XXX, K2D }, \
+    { K30, XXX, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, XXX }, \
+    { K40, K41, K42, XXX, K44, K45, K46, XXX, XXX, K49, K4A, XXX, XXX, XXX }, \
+    { XXX, XXX, XXX, XXX, XXX, XXX, K56, K57, K58, K59, K5A, K5B, K5C, K5D }  \
+}
+
+#define LAYOUT_ansi( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, \
+    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, \
+    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B,      K2D, \
+    K30,      K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C,      \
+    K40, K41, K42,      K44, K45, K46,           K49, K4A,                \
+	                              K56, K57, K58, K59, K5A, K5B, K5C, K5D  \
+) { \
+    { K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D }, \
+    { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D }, \
+    { K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, XXX, K2D }, \
+    { K30, XXX, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, XXX }, \
+    { K40, K41, K42, XXX, K44, K45, K46, XXX, XXX, K49, K4A, XXX, XXX, XXX }, \
+    { XXX, XXX, XXX, XXX, XXX, XXX, K56, K57, K58, K59, K5A, K5B, K5C, K5D }  \
+}
```

Comment:
These layout macros aren't consistent with the layouts shown on your website. Is this one PCB that supports both the layouts shown there:

![](https://wxalbum-10001658.image.myqcloud.com/wxalbum/452092/20191111153422/5d31ad3e80a64189c50a9d5bfbfdafd0.png)
![](https://wxalbum-10001658.image.myqcloud.com/wxalbum/452092/20191111153436/7c03b76aea07f5b0671af6b6c502a321.png)

Reply:
Also, the names do not match between `z70ultra.h` and `info.json`.

Reply:
> These layout macros aren't consistent with the layouts shown on your website. Is this one PCB that supports both the layouts shown there:
> 
> ![](https://camo.githubusercontent.com/d43dfb328ae56d0f08f64a540358b66730d46ecf/68747470733a2f2f7778616c62756d2d31303030313635382e696d6167652e6d7971636c6f75642e636f6d2f7778616c62756d2f3435323039322f32303139313131313135333432322f35643331616433653830613634313839633530613964356266626664616664302e706e67)
> ![](https://camo.githubusercontent.com/1a25c704c26faf015b0404800c61477655b9a030/68747470733a2f2f7778616c62756d2d31303030313635382e696d6167652e6d7971636c6f75642e636f6d2f7778616c62756d2f3435323039322f32303139313131313135333433362f37633033623736616561303766356230363731616636623663353032613332312e706e67)

Yes. the PCB support both of them.

Reply:
> Also, the names do not match between `z70ultra.h` and `info.json`.

recommit it. please check it. Thank you

Reply:
The layouts still do not match. You have `LAYOUT_6.25_spacebar` and `LAYOUT_3_3_split_spacebar` in info.json, and `LAYOUT` and `LAYOUT_ansi` in z70ultra.h.